tests/test_dice_focal_loss.py

```diff
@@ -91,6 +91,28 @@ def test_script(self):
         test_input = torch.ones(2, 1, 8, 8)
         test_script_save(loss, test_input, test_input)
 
+    def test_result_with_alpha(self):
```

Comment:
Can we use `@parameterized.expand` for this test instead of the loops? It reduces the size of the test and breaks them up into separate tests. If this tests only the `alpha` parameter it would be helpful to have fewer such values if they don't really make a large difference, we don't need to test `FocalLoss` itself but just verify `DiceFocalLoss` uses it correctly.